store/localstore/kv.go

```diff
@@ -155,16 +114,11 @@ func (s *dbStore) scheduler() {
 				continue
 			}
 			switch cmd.op {
-			case opSeek:
-				seekCh <- cmd
 			case opCommit:
 				s.doCommit(cmd)
 			}
```

Comment:
I think former implementation doesn't guarantee that, because seek is running at a different goroutine, and there is no lock to prevent that.